release-content/0.16/release-notes/occlusion_culling.md

```diff
@@ -0,0 +1,32 @@
+When it comes to pushing hardware to support larger levels and more detailed meshes, "simply draw less stuff" is a tried-and-true strategy.
+These techniques are collectively referred to as "culling": we can save work whenever we can determine "this doesn't need to be drawn" for cheaper than it would cost to draw the thing despite its pixels later being overwritten completely.
+
+**Occlusion culling** is the idea that we don't need to draw something that's completely blocked by other opaque objects,
+from the perspective of the camera.
+That makes sense! We don't need to draw a car parked on the other side of a building (even if it's within the range used for fustrum culling), and we *never* need to draw all of the teeth inside of a character's closed mouth.
```

Comment:
I don't know if the teeth mentioned here is the best example. Most of the time, they will just be part of the character mesh and unless we are talking about meshlets, it's generally the entire character mesh that is completely visible or not.

Maybe something about vegetation on the other side of a mountain? Or trees occluding other trees in a dense forest?